ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -204,7 +204,7 @@
       + " which are secured with ACLs. Please enable only after careful consideration."
       + " If \"false\", KSQL pull queries will fail against a secure Kafka cluster";
 
-  public static final String KSQL_PULL_QUERIES_ENABLE_CONFIG = "ksql.pull.queries.enable";
+  public static final String KSQL_PULL_QUERIES_ENABLE_CONFIG = "ksql.query.pull.enable";
```

Comment:
Does it matter that the variable names are non consistent with the config name? ("KSQL_PULL_QUERIES_ENABLE_CONFIG").

I assume not, since we don't publish these branches, do we?

Reply:
I think it's ok. The config names are supposed to be hierarchical, but a variable named that way is less readable.